### ISSUE TITLE
V2 output in stdout not formatted correctly when running generate 

### ISSUE THREAD

**Issue by rafcio19** [cloudreach/sceptre#581]
The issue appears when moving from v1 to v2 and affects old `generate-template` / new `generate` command. The output is not formatted correctly, ie. when running `sceptre generate dev/stack.yaml`
```
dict_values(["Outputs:\n  BucketName:\n    Description: Name of S3 bucket to hold website content\n    Value: !Ref 'S3Bucket'\nResources:\n  S3Bucket:\n    Properties:\n      AccessControl: PublicRead\n    Type: AWS::S3::Bucket\n"])
```

Additionally, the command line argument`--output` is not being respected, so `--output json` or `--output yaml` does not vary the output.

I think the problem is in the _write_ function in [helpers.py[(https://github.com/cloudreach/sceptre/blob/master/sceptre/cli/helpers.py#L60), where `output_format` is not being set / respected.

**My setup:**

dev/stack.yaml:
```
---
template_path: demo.py
stack_name: stack
```

demo.py(a copy of troposphere [s3 example](https://github.com/cloudtools/troposphere/blob/master/examples/S3_Bucket.py)):
```
from troposphere import Output, Ref, Template
from troposphere.s3 import Bucket, PublicRead


t = Template()

s3bucket = t.add_resource(Bucket("S3Bucket", AccessControl=PublicRead,))

t.add_output(Output(
    "BucketName",
    Value=Ref(s3bucket),
    Description="Name of S3 bucket to hold website content"
))


def sceptre_handler(sceptre_user_data):
    return t.to_json()
```

OS: `Ubuntu 18.04.1 LTS`

```
$ pip list

Package         Version     Location                                           
--------------- ----------- ---------------------------------------------------
boto3           1.9.73      
botocore        1.12.73     
cfn-flip        1.1.0.post1 
click           6.7         
colorama        0.3.7       
decorator       4.3.0       
docutils        0.14        
Jinja2          2.10        
jmespath        0.9.3       
MarkupSafe      1.1.0       
networkx        2.1         
packaging       16.8        
pip             18.1        
pyparsing       2.3.0       
python-dateutil 2.7.5       
PyYAML          3.13        
s3transfer      0.1.13      
sceptre         2.0.1       
setuptools      40.6.3      
six             1.11.0      
troposphere     2.3.4       
urllib3         1.24.1      
wheel           0.32.3      
```

**Comment by ngfgrant** [comment]
Hi @rafcio19 

I've got a wip for this hope to get something up tomorrow for you to test out. 

Cheers,
Niall

**Comment by rafcio19** [comment]
Thanks but still not there:

```
[
    {
        "Outputs": {
            "BucketName": {
                "Description": "Name of S3 bucket to hold website content",
                "Value": {
                    "Ref": "S3Bucket"
                }
            }
        },
        "Resources": {
            "S3Bucket": {
                "Properties": {
                    "AccessControl": "PublicRead"
                },
                "Type": "AWS::S3::Bucket"
            }
        }
    }
]
```

or in yaml

```
["Outputs:\n  BucketName:\n    Description: Name of S3 bucket to hold website content\n    Value: !Ref 'S3Bucket'\nResources:\n  S3Bucket:\n    Properties:\n      AccessControl: PublicRead\n    Type: AWS::S3::Bucket\n"]
```

It looks like you need to return 1st element from the list to ditch `[ ]` characters for both outputs; replace `\n` with the real carriage return for yaml.

**Comment by ngfgrant** [comment]
@rafco19 I’ve updated the code it has been merged into 2.0.2 branch could you retest from their.
